tmt/steps/provision/testcloud.py

```diff
@@ -468,44 +422,84 @@ def start(self) -> None:
 
         # Create instance
         self.instance_name = self._tmt_name()
+
+        # Prepare DomainConfiguration object before Instance object
+        self._domain = DomainConfiguration(self.instance_name)
```

Comment:
Please, add a declaration similar to `_image` and `_instance`, it's easier to keep track of the attributes of the plugin instance.